internal/app/machined/internal/server/v1alpha1/v1alpha1_server.go

```diff
@@ -67,6 +69,7 @@ const OSPathSeparator = string(os.PathSeparator)
 
 // Server implements the gRPC service server.
 type Server struct {
+	storaged.Server
```

Comment:
I think we can leave this out from here for now. I plan on following up with `storaged` service.

Reply:
ok, so should I leave machine service without this method for now?

Reply:
I would like to share this logic between storage service and maintenance service later on though.

Reply:
yeah, I can make it common to them. It is just that the storage service will be in `storaged` and not in `machined` so we might as well skip in `machined` for now.